package.json

```diff
@@ -11,7 +11,7 @@
     "start": "bash -c 'source start.sh'",
     "dev": "bash -c 'source start.sh'",
     "build": "tsc && vite build",
-    "lint": "eslint",
+    "lint": "eslint --ignore-pattern 'deploy/cdk.out' --ignore-pattern 'src/modules/runs/*'",
```

Comment:
I think we could ignore the `runs` module if it has too much type instability? @raylrui

Reply:
sure, I will have a check again before put it back